GitCommands/Git/GitGpgController.cs

```diff
@@ -103,7 +103,7 @@ public GitGpgController(Func<IGitModule> getModule)
             {
                 CommitStatus cmtStatus;
 
-                string gpg = module.RunGitCmd($"log --pretty=\"format:%G?\" -1 {revision.Guid}");
+                string gpg = module.RunGitCmd($"log --no-show-signature --pretty=\"format:%G?\" -1 {revision.Guid}");
```

Comment:
❓ How is `--no-show-signature` not in conflict with `%G?`?

Reply:
Run ```git log --show-signature --pretty=%G?``` on this repo. 

Now ```git log --show-signature --pretty=%G? --no-show-signature```. Get it now?

Reply:
```git log --pretty='%h %G?' | grep "G"```. Look at all the signed commits in this repo. 

Do a git log on any one of those commits. Look at what adding --show-signature does to it. Our parsing code is not expecting the gpg lines. When you have log.showsignature configured ALL log calls act like show signature is being used.